### ISSUE TITLE
Cannot `:Bundle open` or `:Bopen` gem

### ISSUE THREAD

**Issue by teoljungberg** [tpope/vim-bundler#57]
I can however `% bundle open` from the command line. I wonder if there is a mismatch in how bundler.vim parses the lockfile to get the gems to how bundler itself does it.

Example:

```
:Bundle open redis
Gem "redis" is in bundle but not installed
```

While it does show up in `bundle list`.

```
% bundle list | grep redis
  * fakeredis (0.8.0)
  * redis (4.8.0)
  * redis-gcra (0.5.0)
```

**Comment by teoljungberg** [comment]
Redis is infact installed

```
% bundle check
The Gemfile's dependencies are satisfied
```

And I can `bundle open redis` successfully from my shell.

**Comment by tpope** [comment]
Just pushed fix to #45 might be relevant. Other than that you're going to need to debug `s:project_paths()` on your own. I don't have access to your machine.

**Comment by teoljungberg** [comment]
Nothing immediately stood out while testing the fix for #45.

I wonder about `s:project_paths`, it seems to not be called explicitly anywhere in bundler.vim. Is it called magically by other means?

**Comment by teoljungberg** [comment]
It found the culprit, I had `GEM_PATH` set to a directory that was different than my `GEM_HOME`. And removing the faulty `GEM_PATH` made the problem go away.
